crates/cdk/versions.json

```
@@ -0,0 +1,15 @@
+{
+  "agglayer_image": "ghcr.io/agglayer/agglayer:0.2.0",
+  "cdk_erigon_node_image": "hermeznetwork/cdk-erigon:v2.1.1",
+  "cdk_node_image": "ghcr.io/0xpolygon/cdk:0.4.0-beta1",
+  "cdk_validium_node_image": "0xpolygon/cdk-validium-node:0.7.0-cdk",
+  "zkevm_bridge_proxy_image": "haproxy:3.0-bookworm",
+  "zkevm_bridge_service_image": "hermeznetwork/zkevm-bridge-service:v0.6.0-RC1",
+  "zkevm_bridge_ui_image": "leovct/zkevm-bridge-ui:multi-network-2",
+  "zkevm_contracts_image": "leovct/zkevm-contracts:v8.0.0-rc.4-fork.12",
+  "zkevm_da_image": "0xpolygon/cdk-data-availability:0.0.10",
+  "zkevm_node_image": "hermeznetwork/zkevm-node:v0.7.3",
+  "zkevm_pool_manager_image": "hermeznetwork/zkevm-pool-manager:v0.1.1",
+  "zkevm_prover_image": "hermeznetwork/zkevm-prover:v8.0.0-RC14-fork.12",
+  "zkevm_sequence_sender_image": "hermeznetwork/zkevm-sequence-sender:v0.2.4"
+}
```

Nitpick: add new line to the end.
```suggestion
}

```